navigation/navigation-runtime/src/jsMain/kotlin/androidx/navigation/BrowserHistory.js.kt

```diff
@@ -21,14 +21,19 @@ import org.w3c.dom.Window
 /**
  * Binds the browser window state to the given navigation controller.
  *
+ * If `getBackStackEntryRoute` is null, then:
+ *  1) if a browser url contains a destination route on a start then navigates to destination
+ *  2) if a user puts a new destination route to the browser address field then navigates to the new destination
+ *
+ * If there is a custom `getBackStackEntryRoute` implementation,
+ * then we don't have a knowledge how to parse urls to support direct navigation via browser address input
```

Comment:
Let's suggest to the reads of this KDoc that in such a case they can do it on the app side.